english/fus-roh-dah/fus-roh-dah.md

```diff
@@ -0,0 +1,7 @@
+Wait a second. you're the dragonborn! These things are specs of dust compared to you. In a second you can send them all to oblivion.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.